app/src/main/assets/chart.html

```diff
@@ -190,9 +164,8 @@
     {% endif %}
   {% endfor %}
 
-  $(document).ready(function() {
-    $('#grid').freezeHeader({top: true, left: true});
-  });
+  $('#grid').freezeHeader({top: true, left: true});
```

Comment:
When using jquery to alter DOM is good practice to wrap code inside the document ready event handler like this:
$(document).ready(function(){ 
    $("table").freezeHeader(); 
}); 
This also prevents erros. The problem is that although the script tag is in the bottom of the page, this does not mean that the html is rendered before the script execution. 
So, a good overall practice is always use *$(document).ready(function(){});* for all Javascript code to be executed after page load.